src/block-components/icon/edit.js

```diff
@@ -50,6 +46,10 @@ export const Edit = props => {
 		defaultValue,
 	} = props
 
+	const PremiumColorControls = useMemo( () => applyFilters( 'stackable.block-component.icon.color-controls', null ), [ props ] )
+	const PremiumShapeColorControls = useMemo( () => applyFilters( 'stackable.block-component.icon.shape-color-controls', null ), [ props ] )
+	const PremiumBackgroundShapeControls = useMemo( () => applyFilters( 'stackable.block-component.icon.edit.background-shape', null ), [ props ] )
+
```

Comment:
I believe you can use `[]` as the dependencies here because the filter just returns the component to be rendered.